src/pjs/pjs.proto

```diff
@@ -192,11 +192,30 @@ message ListJobResponse {
   JobInfoDetails details = 3;
 }
 
+enum WalkAlgorithm {
```

Comment:
Good docs.  Document on the line above each entry so they work their way into the API docs.  (proto-docs.md in your PR.)